plugin/evm/validators_cache_test.go

```diff
@@ -0,0 +1,40 @@
+package evm
```

Comment:
Shouldn't this file also test `ValidatorsCache.ByEpoch`?

Reply:
Yes, I am currently working on this, since those function require much more research to do from me.
This PR is created with the work done until now